examples/centralized_design/example.tfvars

```diff
@@ -12,7 +12,7 @@ ssh_key_name = "example-ssh-key" # TODO: update here
 
 ### VPC
 vpcs = {
-  # Do not use `-` in key for VPC as this character is used in concatation of VPC and subnet for module `subnet_set` in `main.tf`
+  # Do not use `-` in key for VPC as this character is used in concatenation of VPC and subnet for module `subnet_set` in `main.tf`
```

Comment:
After merging #49 , this comment can be removed (we don't have anymore `vpc_subnet` keys).
It applies for all examples. Can you removed it, please ?

Reply:
Thank you for the information, I'll remove the comment